platform-hub-api/app/models/docker_repo.rb

```diff
@@ -16,7 +23,7 @@ class DockerRepo < ApplicationRecord
 
   enum status: {
     pending: 'pending',
-    ready: 'ready',
+    active: 'active',
     deleting: 'deleting',
```

Comment:
should this now be `removing` instead, as per the `ACCESS_STATUS` defined above?

Reply:
It's a fair suggestion; my thinking here was that the `DockerRepo` is standalone parent object, which actually does get deleted from the universe, but the individual robot or user within the access policy of the repo just gets removed – i.e. the IAM account may not actually get deleted, so that robot and/or user may not technically be deleted, it's just that they've been removed from this particular repo. Do you think that's enough reason to have this slight difference in wording?

Reply:
Ah I understand, yup I think that makes sense :)